### DISCUSSION TITLE
Web Flux DGS Still uses Tomcat instead of Netty

### DISCUSSION THREAD

**Discussion by praveshg** [Netflix/dgs-framework#399] [Q&A]
On going through the Gettting Started Guide at  https://netflix.github.io/dgs/getting-started/#adding-the-dgs-framework-dependency , I did not see any mention of Spring WebFlux.
So Internally when using **graphql-dgs-webflux-starter**, it is  still using Tomcat Web Container instead of Netty (as used in Spring Web Flux) .
Is this the correct intention , If yes is there any way we can use Netty as underlying Http communication layer instead of Tomcat?

Also apparently I am not able to load http:/localhost:8080/graphiql when using webflux dependency , it works when i use the graphql-dgs-spring-boot-starter . Do we need to do anything else as nothing is mentioned in the Getting Started Guide about this.
I am using Java 11 + Maven + Spring Boot 2.5

**Comment by paulbakker** [comment]
I'm suspecting there is an issue with your dependencies. If you're using Webflux it should be using Netty.
Make sure to use the platform BOM. If you're unable to figure it out, please post your `build.gradle`.
